Sources/TensorFlow/Initializers.swift

```diff
@@ -548,3 +548,40 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
         self = Tensor.glorot(fromStandardNormalScaledBy: normal, shape: shape)
     }
 }
+
+public extension Tensor where Scalar: TensorFlowFloatingPoint {
+    /// Initializer that generates an orthogonal matrix or tensor. 
```

Comment:
Initializer documentation should start with `Creates a ...`. In this case, it should be `Creates an orthogonal matrix or tensor.`